src/stories/pages/ExperimentsPage/styles.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// ExperimentsPage
+
+.experiments__section-header {
+  display: inline-block;
+  margin-right: $spacing-2;
+}
+
+.experiments__ongoing-section {
+  margin-top: $spacing-2;
+  margin-bottom: $spacing-12;
```

Comment:
Ideally pages shouldn't need their own stylesheet. You can apply the margins directly to the Slider component. Also, you can change the `.slider__controls` so it's not floating and add `text-align: right`. This way you don't have to use margin and padding to create more space between sections.